stop self loop with trap handler in sail and fix Priv covergroup selection

Noah-Fotenos opened the pull request.

Noah-Fotenos committed:
stop self loop with trap handler in sail, more error message for covereage, make file naming clearity

jordancarlin commented on framework/src/act/coverreport.py:
```diff
@@ -71,7 +71,7 @@ def report_to_summary(report_path: Path, summary_path: Path) -> None:
 
     if not entries:
-        raise ValueError("No coverage entries found in report.")
+        raise ValueError("No coverage entries found in report " + report_path.as_posix())
```
```suggestion
        raise ValueError(f"No coverage entries found in report {report_path}")
```

Noah-Fotenos committed:
more information header and inprogress fix to vsim command

Noah-Fotenos committed:
revert back non needed changes

Noah-Fotenos committed:
Fix priv to get covereage running on exceptionsM

Files need to be in a folder of with the same name as the file for makefilegen.py to use the correct covergroups.
